base_stage_state/models/base_stage.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# © 2016 Daniel Reis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields, api
+
+
+class BaseStageSet(models.Model):
+    """
+    Organize Stages in Sets
+    """
+    _name = 'base.stage.set'
+    _description = 'Base Stage Set'
+    name = fields.Char(translate=True)
```

Comment:
Maybe a unique validation to avoid naming conflicts between the stage groups? Totally cosmetic, but seems appropriate IMO